mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -218,27 +227,71 @@ def create(self, vals):
 
     @api.multi
     def message_auto_subscribe(self, updated_fields, values=None):
-        """Add the responsible, manager and OpenChatter follow list."""
+        """
+        Add the responsible, manager and author to the OpenChatter follow list
+        """
         self.ensure_one()
         user_ids = [
             self.responsible_user_id.id,
             self.manager_user_id.id,
             self.author_user_id.id,
         ]
         self.message_subscribe_users(user_ids=user_ids, subtype_ids=None)
```

Comment:
@dreispt I have tried to do what you suggested. I still have the same error
That's the error i'm facing RuntimeError: maximum recursion depth exceeded while calling a Python object
I have also anlyse the and i find it corrent from my point of view. Inside the mail.thread class, 
the user_ids is waiting a list of object. If the list is None it builds a list with the id of the current users.

This is why only the current is subscribe to chatter when i deactivate this method. I hop the name of throwing error will be helpfull. It seems that this error is linked to odoo 9